src/3_api.js

```diff
@@ -325,6 +338,12 @@ Server.prototype.XHRRequest = function(url, data, method, storage, callback, nop
  */
 Server.prototype.request = function(resource, data, storage, callback) {
 	var self = this;
+	utils.currentRequestBrttTag = resource.endpoint + '-brtt';
+	if ((resource.endpoint === "/v1/url" || resource.endpoint === "/v1/has-app") && Object.keys(utils.instrumentation).length > 0) {
+		delete utils.instrumentation['-brtt'];
```

Comment:
These are only for link calls, right?

Reply:
We'll add the instrumentation object to /v1/has-app requests and to /v1/url requests.